index.html

```diff
@@ -162,26 +162,38 @@ <h2>Introduction</h2>
   <p><a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a> currently support flat, unstructured strings for accessible names and descriptions.  The result of the name/description computation is thus a flat string. </p>
   <p>The terms "accessible name" and "accessible description" are used to emphasize that they are properties of <a class="termref">accessible objects</a> as exposed by <a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a>.  However, they are frequently referred to hereafter as simply "name" and "description".</p>
 </section>
+<section class="informative" id="terms">
```

Comment:
Any reason why it should be non-normative. Seems fine to define as normative? 
```suggestion
<section id="terms">
```

Reply:
This needs to be a new issue if we are considering this.

Reply:
No problem. We can do that as a followup.